### ISSUE TITLE
Gitlab 9.5 drops support for API V3

### ISSUE THREAD

**Issue by Sudo-Kid** [pyapi-gitlab/pyapi-gitlab#250]
API V3 will be removed in GitLab 9.5, to be released on August 22, 2017. In the 
meantime, we advise you to make any necessary changes to applications that use 
V3. The V3 API documentation is still available.

**Comment by umlaeute** [comment]
GitLab has now dropped support for APIv3 - which makes `pyapi-gitlab` unusable with newer gitlab instances.

**Comment by mattwilmott** [comment]
Yeah just encountered this myself when my Ansible playbooks started failing. sad_panda

**Comment by Sudo-Kid** [comment]
This is a work in progress. I would like to finish it but have been lacking time. Will be trying to get back to development on this project this week or next though.

**Comment by ddj116** [comment]
Just curious if any progress has been made towards supporting the newer Gitlab API.  My team needs this capability and will attempt to implement it if necessary, but if progress has already been made in a fork somewhere, please let me know so we don't re-invent the wheel.

**Comment by umlaeute** [comment]
there's the [`python-gitlab`](https://github.com/python-gitlab/python-gitlab) project, which uses the same module name `gitlab` and allows you to access the APIv4, although the two modules are **not compatible**. (#263).

Following Debian's switch from `pyapi-gitlab` to `python-gitlab` for their `python3-gitlab` package (after an evaluation of the support both projects can currently give), I've personally converted all my consumers of this module to the other one :disappointed:

**Comment by Sudo-Kid** [comment]
I'm still struggling to find time to work on this project. When I took over the project I was using Gitlab at work and this project though shortly after I took it over I was laid off and at my new job we use SVN. This and my hectic work schedule at the moment I'm not sure when I will have time to get back to working on this.

**Comment by ddj116** [comment]
Thanks for the tip, I'll check out `python-gitlab`.  If we end up making progress on `pyapigitlab` I'll try to feed those changes back here via pull request.
